EIPS/eip-4573.md

```diff
@@ -0,0 +1,147 @@
+---
+eip: 4573
+title: Entry Points and Procedures for EVM Code Sections
+description: This proposal introduces EVM procedures as entry points to EOF code sections.
+status: Draft
+type: Standards Track 
+category: Core
+author: Greg Colvin <greg@colvin.org>
```

Comment:
At least one author should have GitHub contact info so the bot can work properly.  Either of the following would be fine in this case (author's choice).
```suggestion
author: Greg Colvin (@gcolvin), Greg Colvin<greg@colvin.org>
```
```suggestion
author: Greg Colvin (@gcolvin)
```